### ISSUE TITLE
Class Training

### ISSUE THREAD

**Issue by xleonplayz** [LILY-QML/LLY-DML#45]
[
[TRAINING.pdf](https://github.com/user-attachments/files/17693320/TRAINING.pdf)
](url)

# Class `Training`

The `Training` class is located in `module/helper` and is responsible for generating the PDF section related to training.

Upon instantiation, the class receives the following information:

- **Names of the Optimizers**
- **Processed Datasets**
- **Structure** - Specifies which features should be included in the PDF, such as:
    - COMPARISON TRAINING MATRIX
    - CONFUSION MATRIX
    - PROBABILITY DISTRIBUTION
    - UNIQUENESS OF STATE
    - STATE INCONSISTENCY REPORT
    - ROC & PRECISION-RECALL
    - LOSS FUNCTIONS OF INPUT DATA
    - COMPARISON BETWEEN OPTIMIZERS

- All data is read from `train.json`.
